nipoppy_cli/nipoppy/data/descriptors/bidscoiner-4.3.2.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "bidscoiner",
+    "description": "bidscoiner",
```

Comment:
Should this (and editor, mapper below) have more detail to explain the purpose?

Reply:
Updated